instagram/src/CommentSection.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+
+export default class CommentSection extends Component {
```

Comment:
It's better to move the `export default` syntax to the bottom of the file. Doing that will save you some issues when working with Redux.